docs/how-to/docker-compose.md

```diff
@@ -27,15 +27,23 @@ flow.to_docker_compose_yaml('docker-compose.yml')
 Jina will generate a `docker-compose.yml` configuration file that you can use directly with 
 `docker-compose` and corresponds to your `Flow`, avoiding the overhead of manually defining all the services needed for the `Flow`.
 
-```{caution}
+````{admonition} Use docker based Executors
+:class: caution
 All Executors in the Flow should be used with `jinahub+docker://...` or `docker://...`.
-```
+````
 
-```{caution}
+````{admonition} Health check available from 3.1.3
+:class: caution
 If you are using Executor which rely on docker image built with a jina version prior to 3.1.3, please remove the 
 health check from the dump yaml file as they are only compatible with 3.1.3+ otherwise your docker compose services will 
 always be `unhealthy`
```

Comment:
```suggestion
If you use Executors that rely on Docker images built with a version of Jina prior to 3.1.3, remove the 
health check from the dumped YAML file, otherwise your Docker Compose services will 
always be "unhealthy."
```